README.md

```diff
@@ -32,6 +32,11 @@ To install jazzy, run `[sudo] gem install jazzy` from your command line.
 Run `jazzy` from your command line. Run `jazzy -h` for a list of additional
 options.
 
+#### Troubleshooting
```

Comment:
Oops, I meant make the emphasized text an h4 header, not the "Troubleshooting" title! Like so:

```md
### Troubleshooting

#### Only extensions are listed in the documentation.

By default, jazzy only documents public declarations. To generate documentation for declarations with a lower accessibility level (internal or private), please set the `--min-acl` flag to `internal` or `private`.
```